front/src/applications/operationalStudies/utils.ts

```diff
@@ -246,3 +246,6 @@ export const formatSpaceTimeData = (
     trainName: trainName || 'Train name not found',
   };
 };
+export const checkNameInvalidity = (name?: string | null) => (name && name.length > 128) || !name;
+export const checkFieldInvalidity = (length: number, field?: string | null) =>
```

Comment:
- As the function returns a boolean, it is better to use a name like nameIsInvalid or stringIsInvalid ;) 
- I think you can move the stringIsInvalid in the file utils/string 
- Also you can rename the param length -> maxLengthAuthorized
```suggestion
export const nameIsInvalid = (name?: string | null) => (name && name.length > 128) || !name;
export const stringIsInvalid = (maxLengthAuthorized: number, field?: string | null) =>
```

Reply:
Done